core/Pos/Binary/Core/Blockchain.hs

```diff
@@ -60,18 +61,16 @@ instance ( Typeable b
               <> encode (T._gbExtra gb)
     decode = do
         enforceSize "GenericBlock" 3
-        header <- decode
-        body   <- decode
-        extra  <- decode
-        toCborError $ T.recreateGenericBlock header body extra
+        _gbHeader <- ({-# SCC "decode_block_header" #-} decode)
```

Comment:
SCC pragmas here also.

Reply:
Do you want me to get rid of them?

Reply:
Well, they are for profiling only. I don't see any reason why would they be in the main code. I may misunderstand something, though.

Reply:
They don't have any runtime overhead if profiling is disabled. But I'll remove them anyway, since they're better placed in the `Bi` instances of the particular types.